cobigen/cobigen-tsplugin/pom.xml

```diff
@@ -26,12 +26,12 @@
     <dependency>
       <groupId>com.devonfw.cobigen</groupId>
       <artifactId>core-api</artifactId>
-      <version>5.4.0</version>
+      <version>6.2.0</version>
     </dependency>
     <dependency>
       <groupId>com.devonfw.cobigen</groupId>
       <artifactId>core</artifactId>
-      <version>[5.4.0,)</version>
+      <version>6.2.0</version>
```

Comment:
btw. is it necessary to increase the version?
Just asking, as I would make the tsplugin incompatible with older versions of cobigen core.

Reply:
The v6.2.0 contains the fix for update nestserver since the implemenation of ExternalServer lies there. What do you mean with making it incompatible instead of increasing the version? Should they be the same thing?